src/site/site-assets/icons/README.md

```diff
@@ -0,0 +1,25 @@
+# Your Favicon Package
```

Comment:
What is this file and do we need it?

Reply:
No, it should have been deleted. I'll remove it in the next commit.